OpenSim/Tools/CMC.h

```diff
@@ -113,11 +113,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(CMC, TrackingController);
     /** Whether or not to check the target time. */
     bool _checkTargetTime;
     /** Storage object for the position errors. */
-    Storage *_pErrStore;
+    std::shared_ptr<Storage> _pErrStore;
     /** Storage object for the velocity errors. */
-    Storage *_vErrStore;
+    std::shared_ptr<Storage> _vErrStore;
     /** Storage object for the stress term weight. */
-    Storage *_stressTermWeightStore;
+    std::shared_ptr<Storage> _stressTermWeightStore;
```

Comment:
As per original version, the copy constructor of this class intends to do *shallow copy*. `shared_ptr` suits the need I think.